handlers/forum/customindex.go

```diff
@@ -69,9 +69,13 @@ func ForumCustomIndexItems(cd *common.CoreData, r *http.Request) []common.IndexI
 			})
 		}
 		if tid, err := strconv.Atoi(topicID); err == nil && cd.HasGrant("forum", "topic", "post", int32(tid)) {
+			name := "New Thread"
+			if strings.HasPrefix(r.URL.Path, "/private") {
```

Comment:
@jules doesn't this code base already have functions around detecting this? We might need to enhance them but we should use it here.